<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle"><string>Gestion Réservations - Tableau de bord</string></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="sideMenu">
      <property name="minimumWidth"><number>180</number></property>
      <layout class="QVBoxLayout" name="sideLayout">
       <item><widget class="QLabel" name="lblLogo"><property name="text"><string>Compagnie Aérienne</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
       <item><widget class="QPushButton" name="btnDashboard"><property name="text"><string>Tableau de bord</string></property></widget></item>
       <item><widget class="QPushButton" name="btnClients"><property name="text"><string>Clients</string></property></widget></item>
       <item><widget class="QPushButton" name="btnVols"><property name="text"><string>Vols</string></property></widget></item>
       <item><widget class="QPushButton" name="btnReservations"><property name="text"><string>Réservations</string></property></widget></item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation"><enum>Qt::Vertical</enum></property>
         <property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
        </spacer>
       </item>
       <item><widget class="QPushButton" name="btnLogout"><property name="text"><string>Déconnexion</string></property></widget></item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="pageDashboard">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item><widget class="QLabel" name="lblDashboardTitle"><property name="text"><string>Tableau de bord</string></property></widget></item>
        <item>
         <layout class="QHBoxLayout" name="statsLayout">
          <item><widget class="QLabel" name="lblTotalVols"><property name="text"><string>Vols: 0</string></property></widget></item>
          <item><widget class="QLabel" name="lblTotalClients"><property name="text"><string>Clients: 0</string></property></widget></item>
          <item><widget class="QLabel" name="lblTotalReservations"><property name="text"><string>Réservations: 0</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageClients">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="clientsActions">
          <item><widget class="QLineEdit" name="lineClientSearch"><property name="placeholderText"><string>Rechercher un client...</string></property></widget></item>
          <item><widget class="QPushButton" name="btnClientAdd"><property name="text"><string>Ajouter</string></property></widget></item>
          <item><widget class="QPushButton" name="btnClientEdit"><property name="text"><string>Modifier</string></property></widget></item>
          <item><widget class="QPushButton" name="btnClientDelete"><property name="text"><string>Supprimer</string></property></widget></item>
         </layout>
        </item>
        <item><widget class="QTableWidget" name="tableClients"/></item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageVols">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="volsActions">
          <item><widget class="QLineEdit" name="lineVolSearch"><property name="placeholderText"><string>Rechercher un vol...</string></property></widget></item>
          <item><widget class="QPushButton" name="btnVolAdd"><property name="text"><string>Ajouter</string></property></widget></item>
          <item><widget class="QPushButton" name="btnVolEdit"><property name="text"><string>Modifier</string></property></widget></item>
          <item><widget class="QPushButton" name="btnVolDelete"><property name="text"><string>Supprimer</string></property></widget></item>
         </layout>
        </item>
        <item><widget class="QTableWidget" name="tableVols"/></item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageReservations">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="reservActions">
          <item><widget class="QPushButton" name="btnResAdd"><property name="text"><string>Nouvelle réservation</string></property></widget></item>
          <item><widget class="QPushButton" name="btnResEdit"><property name="text"><string>Modifier</string></property></widget></item>
          <item><widget class="QPushButton" name="btnResDelete"><property name="text"><string>Supprimer</string></property></widget></item>
          <item><widget class="QPushButton" name="btnPrintTicket"><property name="text"><string>Générer ticket</string></property></widget></item>
         </layout>
        </item>
        <item><widget class="QTableWidget" name="tableReservations"/></item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>